docs/api/api.yml

```diff
@@ -386,29 +386,32 @@ paths:
     post:
       tags:
       - directory
-      summary: Adds a file or a subdirectory.
+      summary: Adds one or more files.
       description: |
-        Sub request of a request with `Content-Type` value `multipart/form-data`.
-        By default the files listed in the body of the main request are saved in `dir_path` under the file name provided
-        in the `Content-Disposition` header. However, if a file has the extension ZIP, it
-        is extracted in `dir_path` and `Content-Disposition` is ignored (and may be left out).
-        If `dir_path`, or any of its parents do not exist yet, they are first created.
+        The `Content-Type` must always be `multipart/form-data`, which means the body is formatted using the
```

Comment:
What type of response code do you want for other content types? #63 is causing 'not-implemented'

Reply:
The proper response is 406 Not acceptable. "The requested resource is capable of generating only content not acceptable according to the Accept headers sent in the request." See [the wikipedia page on http status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors).

Reply:
I think 400 Bad request is the best option here.

Reply:
Seeing https://github.com/scalatra/scalatra/blob/2a038af00bea06e1bc97e328be92ad9ec7ccf3b5/core/src/main/scala/org/scalatra/servlet/FileUploadSupport.scala#L101

I guess this should be: `Content-Type` must always start with `multipart/`

Reply:
I think is must be `multipart/form-data` specifically. There are other types of `multipart`, see https://tools.ietf.org/html/rfc2046.html#section-5.1. I am not sure if we can or need to handle those as well. I specified it so that only `multipart/form-data` is supported, but if you find something in the document I mentioned above that makes you think we should support more than that, let me know.